src/dodal/devices/qbpm1.py

```diff
@@ -3,4 +3,9 @@
 
 
 class QBPM1(Device):
+    """Quadrant Beam Position Monitor"""
+
     intensity: EpicsSignalRO = Cpt(EpicsSignalRO, "-DI-QBPM-01:INTEN", kind=Kind.normal)
+    intensityC: EpicsSignalRO = Cpt(
```

Comment:
Should: Could we have a more descriptive name than `IntensityC`?

Reply:
No! I would probably have changed but honestly I don't know what this PV actually should be called. I believe it's some kind of aggregate measurement, but then if that's the case I don't know what `intensity` is measuring. In any case, it's no longer needed so I will remove it.

Reply:
Remove it is best. If something like this comes up in the future we should make the effort to understand it (by discussion with Neil or controls) then put in a more sensible name. Part of the aim of this project is to get a better understanding of what we're actually doing